tests/tests/test_storage.py

```diff
@@ -68,8 +70,10 @@ def test_post_process(self):
         self.assertTrue(('screen.css', 'screen.css', True) in processed_files)
         self.assertTrue(('scripts.js', 'scripts.js', True) in processed_files)
 
+    @unittest.skip("don't care")
```

Comment:
Doesn't this test a condition we'll soon have, with the s3 boto storage (which doesn't have a path method implemented). Was this disabled because it's failing, or for some other reason?

Reply:
Or, if it's passing in travis but failing locally, I can probably help with the configuration. I actually wrote these tests; they were merged them into upstream a while back (jazzband/django-pipeline#531)